desktop-widgets/tab-widgets/TabDiveStatistics.cpp

```diff
@@ -69,8 +69,11 @@ void TabDiveStatistics::updateData()
 	else
 		ui->sacLimits->setAverage("");
 
-	ui->tempLimits->setMaximum(get_temperature_string(stats_selection.max_temp, true));
-	ui->tempLimits->setMinimum(get_temperature_string(stats_selection.min_temp, true));
+	if (stats_selection.combined_count > 1){
```

Comment:
What happens if the selection is changed from multiple to one dive? Then the old values remain, no?

Unless I'm missing something, I suggest adding an else branch that clears the fields in the case of a single dive.

Reply:
When you change from multiple to one dive, the new value is the only which is shown,